### PR TITLE
Succeed CLA status for excluded user.

### PR DESCRIPTION
**Pull request by metanerd**
#### Summary

CLA github status will still be on pending even when excluded bot user. Succeeding the CLA github status is necessary for an excluded user.

### CODE REVIEW

File: server/cla.go [hunk 1]
```diff
@@ -52,7 +52,14 @@ func (s *Server) checkCLA(pr *model.PullRequest) {
...
 
 	if contains(s.Config.CLAExclusionsList, username) {
-		mlog.Info(fmt.Sprintf("%s is excluded to sign the CLA", username))
+		status := &github.RepoStatus{
+			State:       github.String(stateSuccess),
+			Description: github.String(fmt.Sprintf("%s excluded", username)),
+			TargetURL:   github.String(s.Config.SignedCLAURL),
+			Context:     github.String(s.Config.CLAGithubStatusContext),
+		}
+		mlog.Debug("will succeed CLA status for excluded user", mlog.String("user", username))
```
Comment (khos2ow) [inline server/cla.go:61]: Why did you change it from `Info` to `Debug` though?

Comment (metanerd) [reply server/cla.go:61]: To reduce at some point the noise level in the logs. You want to keep `Info`?

Comment (khos2ow) [reply server/cla.go:61]: No, just wondering. Looks good with debug.
